packages/cashier/src/Containers/withdrawal.jsx

```diff
@@ -58,8 +59,10 @@ const Withdrawal = ({
 
     React.useEffect(() => {
         if ((iframe_url || verification_code) && isDesktop()) {
-            if (/^(UST|eUSDT)$/i.test(currency) && typeof setSideNotes === 'function') {
+            if (/^(UST)$/i.test(currency) && typeof setSideNotes === 'function') {
```

Comment:
We can simplify this whole condition block by checking if setSideNotes is a function first, then if it's UST or eUSDT inside, instead of checking if it's a function in multiple places.

Reply:
Yup that makes complete sense...thanks for highlighting it 🙏